src/sentry/filestore/gcs.py

```diff
@@ -150,7 +151,7 @@ def size(self):
     def _get_file(self):
         def _try_download():
             self.blob.download_to_file(self._file)
-            self.file.seek(0)
+            self._file.seek(0)
```

Comment:
Might have been a typo in https://github.com/getsentry/sentry/pull/10717/files
@mitsuhiko can you confirm?

Reply:
indeed.